test/integration/users.test.js

```diff
@@ -186,6 +186,68 @@ describe("Users", function () {
     });
   });
 
+  describe("PATCH /add-default-status", function (done) {
+    beforeEach(async function () {
+      await addUser(userData[1]); // status does not exists user
+      await addUser(userData[0]); // status active user
+    });
+
+    it("Should add default status to user where status does not exists", function (done) {
+      chai
+        .request(app)
+        .patch(`/users/add-default-status`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a("object");
+          expect(res.body.message).to.equal("Default status added to users successfully!");
+          expect(res.body.addedDefaultStatusUsers).to.include.members(["nikhil"]);
+          expect(res.body.addedDefaultStatusUsers).to.not.include.members(["ankur"]);
```

Comment:
Use the same user objects here which were added in firestore in the beforeEach block.
It'll be more readable that way

Reference: https://github.com/Real-Dev-Squad/website-backend/pull/654/files#diff-a07acf94413fffcf0e2831b131eb44a057a498ed297a72190c8f23fc6561b50e